packages/api/test/util/SingleAccountSigner.ts

```diff
@@ -19,16 +19,32 @@ export default class SingleAccountSigner {
     this.signDelay = signDelay;
   }
 
-  public async sign (extrinsic: Extrinsic, address: string, options: SignatureOptions): Promise<number> {
-    if (!this.keyringPair || String(address) !== this.keyringPair.address) {
+  // @deprecated Kept here until we have it removed completely
```

Comment:
I would propose to avoid committing commented code (like the rust guys do). This code could be just deleted

Reply:
I'll add a TODO - I prefer having it there before the deprecated interface is removed.